pinc/ProjectSearchForm.inc

```diff
@@ -142,12 +131,51 @@ class ProjectSearchWidget
             elseif ( $comparator == 'LIKE' )
             {
                 $value = normalize_whitespace($value);
-
                 $contribution = "$column_name LIKE '%$value%'";
             }
         }
         return $contribution;
     }
+
+    function sql_from_multi($values, $column_name, $comparator)
```

Comment:
I would be explicit in the variable name at least that you expect $values to be encoded already. Something like ```$encodedValues``` so people know if you need to encode them or not before calling this method.

Reply:
I would rather move the encoding to this function.

Reply:
Done this.